test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -42,6 +43,12 @@ boostIntGenerator RandomCode::randOpLengGen = boostIntGenerator(gen, opLengDist)
 boostIntGenerator RandomCode::randUniIntGen = boostIntGenerator(gen, uniIntDist);
 boostUInt64Generator RandomCode::randUInt64Gen = boostUInt64Generator(gen, uInt64Dist);
 
+const std::vector<eth::Instruction> RandomCode::invalidOpcodes = boost::assign::list_of(eth::Instruction::INVALID) \
```

Comment:
Why not just initialize with an initializer list in braces?
`const std::vector<eth::Instruction> RandomCode::invalidOpcodes{eth::Instruction::INVALID, eth::Instruction::PUSHC, ...};`